unlock-app/src/stories/creator/DemoPage.stories.js

```diff
@@ -31,10 +31,53 @@ const store = createUnlockStore({
   currency: {
     USD: 195.99,
   },
-})
+}
+const unlockedState = {
+  ...lockedState,
+  account: {
+    address: '0xAaAdEED4c0B861cB36f4cE006a9C90BA2E43fdc2',
+    balance: '989898989898',
+  },
+  keys: {
+    '0xab7c74abc0c4d48d1bdad5dcb26153fc87eeeeee': {
+      lock: '0xab7c74abc0c4d48d1bdad5dcb26153fc8780f83e',
+      owner: '0xAaAdEED4c0B861cB36f4cE006a9C90BA2E43fdc2',
+      expiration: new Date('December 31, 3000 12:00:00').getTime() / 1000,
+      transaction:
+        '0x06094207a34b7f1c19b93d337d0c91c357d45ff8e584deb003e67b637db3d876',
+    },
+  },
+  transactions: {
```

Comment:
Do we need the transactions?

Reply:
yes. Otherwise the lock is not confirmed, paywall shows instead of the flag